rs/state_machine_tests/src/lib.rs

```diff
@@ -928,7 +928,7 @@ impl StateMachineBuilder {
             config: None,
             checkpoint_interval_length: None,
             subnet_type: SubnetType::System,
-            enable_canister_snapshots: false,
+            enable_canister_snapshots: true,
```

Comment:
Does it even make sense to keep this flag around now that the feature has been released?

Reply:
The existing tests all either leave it at default or explicitly set it to true. I guess the question is if there should be tests that explicitly set it to false. If not, getting rid of the flag might be best.

Reply:
We typically get rid of the flag once the feature has been deployed to production and some time has passed to consider it stable (see also my other reply below).